core/src/main/scala/com/salesforce/op/stages/impl/feature/MultiPickListMapVectorizer.scala

```diff
@@ -63,9 +68,18 @@ class MultiPickListMapVectorizer[T <: OPMap[Set[String]]]
         k -> cats.map(_ -> 1L).groupBy(_._1).map { case (c, a) => c -> a.map(_._2).sum }
       }
     }
+    implicit val classTag: ClassTag[T#Value] = ReflectionUtils.classTagForWeakTypeTag[T#Value]
+    implicit val spark = dataset.sparkSession
+    implicit val kryo = new KryoSerializer(spark.sparkContext.getConf)
+    val uniqueCounts = countMapUniques(dataset, size = inN.length, bits = $(bits))
+    val n = dataset.count()
```

Comment:
can you calculate the total size when you get the unique counts and save a pass through the data?